lib/diagnostic.rb

```diff
@@ -115,10 +120,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = Hash.new('')'
```

Comment:
Close, the `person` array should have been instantiated with two keys, `:first_name` and `:age`, so 

`person = {first_name: "Jordan", age: 88}`